WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -307,6 +314,9 @@ extension WPStyleGuide {
         fileprivate static let snippetParagraph         = NSMutableParagraphStyle(
             minLineHeight: snippetLineSize, lineBreakMode: .byWordWrapping, alignment: .natural
         )
+        fileprivate static let snippetHeaderParagraph         = NSMutableParagraphStyle(
```

Comment:
Sticking with the coding style of the file, the `=` should be balanced with the others
```suggestion
        fileprivate static let snippetHeaderParagraph   = NSMutableParagraphStyle(
```